### PR TITLE
42 - PostLike Prisma update + post API route

### PR DESCRIPTION
**Pull request by therearedoors**
- updated Prisma model to include PostLike, as two one-to-many relations on Users and Posts
- added an API route post/:postId/like and a controller for creating likes

### CODE REVIEW

File: src/controllers/post.js [hunk 1]
```diff
@@ -73,6 +73,35 @@ export const createComment = async (req, res) => {
...
+    await dbClient.postLike.create({
+      data: {
+        post: {
+          connect: {
+            id: parseInt(postId)
+          }
+        },
+        user: {
+          connect: {
+            id: 1
```
[the remark this answers is not in the document]
Comment (therearedoors) [reply src/controllers/post.js:94]: Was a placeholder for testing the route - I'll update this now

File: src/controllers/post.js [hunk 2]
```diff
@@ -73,6 +73,36 @@ export const createComment = async (req, res) => {
   }
 }
 
+export const likePost = async (req, res) => {
+  const { postId } = req.params
+  const { id } = req.user
+  const likeOnPost = await dbClient.post.findUnique({
```
Comment (Auenc) [inline src/controllers/post.js:79]: This should be within the try catch as well

**Review by Auenc — CHANGES_REQUESTED** [review]
All looks good to me, just move that prisma call to be inside the try/catch, and it'll be ready to be merged!

**Review by nicopicchio — APPROVED** [review]
Happy with changes

**Pull request merged**